pkg/ExternalPackages/versions.props

```diff
@@ -2,19 +2,28 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
   <ItemGroup>
-    <ExternalPackage Include="runtime.win7-amd64.runtime.native.System.Data.SqlClient.sni">
+    <ExternalPackage Include="runtime.win7-x64.runtime.native.System.Data.SqlClient.sni">
```

Comment:
I believe this is used when building tests against packages.  @chcosta can provide more details.